src/traces/heatmap/hover.js

```diff
@@ -46,8 +48,8 @@ module.exports = function hoverPoints(pointData, xval, yval, hovermode, contour)
             return;
         }
     }
-    else if(Fx.inbox(xval - x[0], xval - x[x.length - 1]) > Fx.MAXDIST ||
-            Fx.inbox(yval - y[0], yval - y[y.length - 1]) > Fx.MAXDIST) {
+    else if(Fx.inbox(xval - x[0], xval - x[x.length - 1]) > MAXDIST ||
+            Fx.inbox(yval - y[0], yval - y[y.length - 1]) > MAXDIST) {
```

Comment:
This bug was pretty painful to track down.

It was introduced in https://github.com/plotly/plotly.js/pull/154/commits/d7abd18f92a8c28217ac9fa14a777e5171d99254 when some `graph_interact.js` constants got moved to `plots/cartesian/constants.js`. 

Unfortunately, `Fx.MAXDIST` was still being used in heatmap hover which was `undefined`.

**Reminder:** don't forgot to grep/ack/ag when moving things around!